pkg/tasks/tasks.go

```diff
@@ -107,6 +107,13 @@ func WithFullInstall(t Tasks) Tasks {
 	return WithBinariesOnly(t).
 		append(kubernetesConfigFiles()...).
 		append(Tasks{
+			{
+				Fn: func(s *state.State) error {
+					s.Logger.Infoln("Disable nm-cloud-setup on aws rhel nodes...")
+					return s.RunTaskOnLeader(disableNMCloudSetup)
+				},
+				ErrMsg: "failed to disable nm-cloud-setup",
+			},
```

Comment:
As discussed earlier, I'd run this at the very early beginning, even before installing binaries.

Reply:
@moadqassem Seems like this is a leftover that should be removed.